pkg/webhook/server.go

```diff
@@ -172,3 +243,15 @@ func (s *Server) InjectFunc(f inject.Func) error {
 	s.setFields = f
 	return nil
 }
+
+func generatePath(gvk schema.GroupVersionKind) string {
+	var pathItems []string
+	splittedGroup := strings.Split(gvk.Group, ".")
```

Comment:
why aren't we just `strings.Replace(gvk.Group, ".", "-")` or something?

Reply:
Curret behavior doesn't include domain name in the path:
```
Group: apps.example.com
Version: v1
Kind: Foo
```
I will get `/mutate-apps-v1-foo` instead of `/mutate-apps-example-com-v1-foo`. IMO it's more concise, but there may be a small risk of collision. e.g. with 
```
Group: apps.bar.com
Version: v1
Kind: Foo
```

I don't have a strong opinion about if we should have domain name.
Thoughts?

Reply:
I don't think conciseness here is valuable (only ever have to write this once, and it's usually going to be autogenerated), and I'd rather avoid collisions